### ISSUE TITLE
Missing docker-for-mac.ovpn 

### ISSUE THREAD

**Issue by partizanos** [wojas/docker-mac-network#9]
Hello I am not sure where docker-for-mac.ovpn file is generated.
Is there any step/configuration I am missing?

**Comment by wojas** [comment]
It will be generated in the root folder of this repository. Note that this may take about 30 seconds, as it needs to generate keys.

If this does not work, please check the output of `docker-compose logs` for any errors.

**Comment by rfer** [comment]
I'm facing a similar issue. I got this working before, but "suddenly" I started to get the following:

```
openvpn_1  | WARNING!!!
openvpn_1  |
openvpn_1  | You are about to remove the EASYRSA_PKI at: /etc/openvpn/pki
openvpn_1  | and initialize a fresh PKI here.
openvpn_1  |
openvpn_1  | Type the word 'yes' to continue, or any other input to abort.
openvpn_1  |   Confirm removal:
openvpn_1  | Aborting without confirmation.
dockermacnetwork_openvpn_1 exited with code 9

openvpn_1  | *** REGENERATING ALL CONFIGS ***
```

Any hints on what this might be?

**Comment by wojas** [comment]
@rfer I'm not sure about the confirmation, but the configs will be regenerated if `docker-for-mac.ovpn` is missing. 

You can manually delete `config/*`, which will at least work around the confirmation error.

I guess `helpers/run.sh` should delete those files. It looks like I commented out `rm -rf /etc/openvpn/*`, but I don't remember why.

**Comment by rfer** [comment]
@wojas Yeah, eventually I tried removing the commented line and it worked. Was about to ask you why it was commented, as I don't see any harm with it.
